cristin-organization/src/main/java/no/unit/nva/cristin/organization/HttpExecutorImpl.java

```diff
@@ -96,20 +100,41 @@ protected SubSubUnitDto fetch(URI uri) throws InterruptedException, NotFoundExce
         }
     }
 
+    protected SearchResponse<Organization> query(URI uri) throws NotFoundException, FailedHttpRequestException {
+        HttpRequest httpRequest = createHttpRequest(addLanguage(uri));
+        HttpResponse<String> response = sendRequest(httpRequest);
+        if (isSuccessful(response.statusCode())) {
+            try {
+                List<Organization> organizations = getOrganizations(response);
+                final int count =  response.headers().firstValue("X-Total-Count").isPresent()
+                        ? Integer.parseInt(response.headers().firstValue("X-Total-Count").get())
+                        : organizations.size();
+                return new SearchResponse<Organization>(uri).withHits(organizations).withSize(count);
```

Comment:
Should `uri` be Cristin uri or NVA uri?

Reply:
Uri is overwritten with .setId(...) later... Id is against NVA API